grpc-native/src/main/java/org/ballerinalang/net/grpc/callback/StreamingCallableUnitCallBack.java

```diff
@@ -95,7 +98,12 @@ public ReturnStreamUnitCallBack(Runtime runtime, StreamObserver requestSender,
         @Override
         public void notifySuccess(Object response) {
             if (response != null) {
-                requestSender.onNext(new Message(this.outputType.getName(), response));
+                if (isRecordMapValue(response)) {
+                    requestSender.onNext(new Message(this.outputType.getName(),
+                            ((BMap) response).get(StringUtils.fromString("value"))));
```

Comment:
```suggestion
                            ((BMap) response).get(StringUtils.fromString("content"))));
```

Reply:
@BuddhiWathsala the field name should be `content` right?

Reply:
In streaming cases, the records reach the `onSuccess` call as a `BMap`. The key that record is the `value`.